$OpenBSD: patch-desktopwidget_cpp,v 1.1 2018/12/17 18:49:50 jca Exp $
Fix for ports-gcc-4.9:
desktopwidget.cpp:34:5: error: '::exit' has not been declared
Index: desktopwidget.cpp
--- desktopwidget.cpp.orig
+++ desktopwidget.cpp
@@ -31,7 +31,7 @@ DesktopWidget::DesktopWidget(QWidget *parent, const ch
   if(!currentImage)
   {
     fprintf(stderr, "Not enough memory!\n");
-    ::exit(1);
+    std::exit(1);
   }
 //  fprintf(stderr, "Desktop size: %dx%d\n",width(), height());
 }
